errors/next-script-for-ga.md

```diff
@@ -0,0 +1,70 @@
+# Next Script for Google Analytics
+
+### Why This Error Occurred
+
+An inline script was used for Google analytics which might impact your webpage's performance.
+
+### Possible Ways to Fix It
+
+#### Script component (experimental)
+
+Use the Script component with the right loading strategy to defer loading of the script until necessary.
+
+```jsx
+import Script from 'next/experimental-script'
+
+const Home = () => {
+  return (
+    <div class="container">
+      <Script>
+        {`
```

Comment:
Quick question for @janicklas-ralph just to make sure, doesn't this cause React to run the XSS protection causing certain cases to be escaped where we wouldn't want it to? That's the reason it's passed as `dangerouslySetInnerHTML`

Reply:
Good point. We support both ways [here](https://github.com/vercel/next.js/blob/canary/packages/next/client/experimental-script.tsx#L68-L79)

Since we pass as a string I'm not sure if the content is sanitized?

Reply:
@timneutkens This thread seems to be unrelated to the PR anymore. Could you resolve it?

Reply:
It looks like we use `el.textContent` when passing the script content as a child [here](https://github.com/vercel/next.js/blob/d2551bbbc7588d392fd47420c4eea803794f967f/packages/next/client/script.tsx#L79-L90) which does not apply react's HTML entities escaping so this does look like it can be resolved.